pycaching/cache.py

```diff
@@ -1020,7 +1022,7 @@ def _get_log_counts_from_print_page(soup):
         if not element:
             raise errors.ValueError("Log counts could not be found.")
 
-        # Text gives numbers and verbose descriptions of the current values as well as an
+        # Text gives numbers and verbose s of the current values as well as an
```

Comment:
There is a typo.

Reply:
I don’t understand — I did not alter line 1020 - 1025.

Reply:
You can see from the referenced diff that `descriptions` has been there before which still should be kept.

Reply:
Fixed it!